components/trips/TripStopNameForm.tsx

```
@@ -0,0 +1,76 @@
+
+
+    return (
+        <View className="w-full">
+            <View className='mb-4'>
+                <Text className='dark:text-white font-bold text-lg'>
+                    {tripStop ? "Modifier" : "Ajouter"} une étape
+                </Text>
+            </View>
+            <View>
+                <Text className='font-bold ml-2 dark:text-white text-sm'>
+                    Nom de l'étape*
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)